ci-operator/config/openshift/ansible-operator-plugins/openshift-ansible-operator-plugins-main.yaml

```diff
@@ -99,6 +117,11 @@ tests:
   container:
     from: get-requirements
   run_if_changed: ^(images/ansible-operator/Pipfile*|openshift/install-ansible.sh|openshift/requirements*.txt|openshift/Dockerfile.requirements)$
+- as: verify-collections
+  commands: |
+    git diff --exit-code
```

Comment:
nit: I think we can add some `echo` command to have a logline inside `artifacts/test/build-log.txt`; otherwise, this file is mostly blank and might confuse whether the test would have actually run or not. 

Could you please also add the same for `verify-requirements` as well, since it would have also faced the issue?

Reply:
`echo Verify ansible-collections: running git diff to detect changes...` or something like that.